tests/foreman/api/test_host.py

```diff
@@ -325,19 +327,19 @@ def test_positive_create_and_update_with_puppet_proxy(default_sat):
     :CaseImportance: Critical
     """
     # TODO Define the default capsule/SP port + URL on hosts.Capsule
-    proxy = default_sat.api.SmartProxy().search(query={'search': f'url = {default_sat.url}:9090'})[
-        0
-    ]
-    host = default_sat.api.Host(puppet_proxy=proxy).create()
+    proxy = module_puppet_enabled_sat.api.SmartProxy().search(
+        query={'search': f'url = {module_puppet_enabled_sat.url}:9090'}
+    )[0]
```

Comment:
This should be `module_default_puppet_proxy` that you have created in this same PR!